dtcli/scripts/dt.py

```diff
@@ -254,7 +280,37 @@ def genca(**kwargs):
     )
 
 
-_deprecate_above, _deprecate_below = deprecated("dt ext generate-developer-pem")
+@typer_extension.command()
+def generate_certificate_authority(
+    ca_cert: str = typer.Option(
```

Comment:
_genca() expects ca_cert_path and ca_key_path to be paths/strings with paths to files not folders. We could do something like -o "Path to folder", then in generate_certificate_authority() add default names (ca.key, ca.pem), but this would mean that users wouldn't be able to generate files with custom names. Would it be preferable?

Reply:
opennssl does it:
-keyout key.pem -out cert.pem

Reply:
Nah, you're right. Can we do both? So: if the out is a directory - use default name. Else: require both paths to be set.